terraform/outputs.tf

```diff
@@ -71,6 +71,11 @@ output "bigquery_unique_events_view_id" {
   value       = google_bigquery_table.unique_events_view.table_id
 }
 
+output "bigquery_commit_review_status_table_id" {
+  description = "BigQuery commit_review_status table resource."
+  value       = var.commit_review_status.enabled ? module.commit_review_status.commit_review_status_table_id : null
```

Comment:
Because the module uses the `count` meta argument [here](https://github.com/abcxyz/github-metrics-aggregator/blob/main/terraform/bigquery.tf#L526), you will need to add an index value here.

```suggestion
  value       = var.commit_review_status.enabled ? module.commit_review_status[0].commit_review_status_table_id : null
```

**nit** Additionally, because the module depends on the enabled flag, I think you could even "simplify" this to
```suggestion
  value       = try(module.commit_review_status[0].commit_review_status_table_id, null)
```